developer/src/kmc-package/src/compiler/kmp-compiler.ts

```diff
@@ -71,7 +71,11 @@ export class KmpCompiler {
     if(kps.options) {
       for (let [src,dst] of keys) {
         if (kps.options[src]) {
-          kmp.options[dst] = kps.options[src];
+          if(dst == 'graphicFile' || dst == 'readmeFile') {
+            kmp.options[dst] = /[/\\]?([^/\\]*)$/.exec(kps.options[src])[1];
```

Comment:
So... why are we stripping the path selectively here?  I assume there are other `dst` values that should remain unstripped?

Reply:
Those are the only two options that are paths